src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -575,11 +575,11 @@ def create_table_statement(
     schema: str,
     replace: bool = False,
     error: bool = True,
-    temp: bool = False,
+    table_type: str = "",
```

Comment:
```suggestion
    table_type: str = EMPTY_STRING,
```
the same below

Reply:
Changed for the internal usage in `analyzer_utils.py` and `snowflake_plan_node.py`. I assumed we don't want to use this as the default value for public API in `dataframe_writer` :)

Reply:
Hmmm not able to change `snowflake_plan_node.py` due to circular dependency LOL

Reply:
I see, yeah I think just changing it at analyzer_utils.py should be fine because in this file we are generally using `EMPTY_STRING` instead of `""`.